README.adoc

```diff
@@ -72,15 +76,16 @@ fragment menuFields on MenuItem {
 }
 
 query {
-    guillotine(project: "hmdb", branch: "draft") {
-        getMenu(contentKey: "/hmdb/movies")  {
-            menuItems {
-                ...menuFields
-            }
-        }
+    guillotine {
+       menu  {
+           menuItems {
+             ...menuFields
+          }
+       }
     }
 }
 ```
 
-In this example, we are querying the menu for the content with the key `/hmdb/movies`.
+If the `X-Guillotine-SiteKey` header is set, the menu will be retrieved for the given site. Otherwise, the menu will be retrieved for the root content with path `/`.
```

Comment:
```suggestion
If the `X-Guillotine-SiteKey` header is set, the menu will be retrieved for the given site. Otherwise, the menu will be retrieved for project.
```